google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1791,6 +1792,10 @@ public CallCredentialsProvider getCallCredentialsProvider() {
     return callCredentialsProvider;
   }
 
+  public boolean usesNoCredentials() {
```

Comment:
Does it really have to be public? one usage is within this file itself. Can we make it private?